docs/development.md

```diff
@@ -56,7 +56,7 @@ $ oc new-project managed-upgrade-operator
 * Run the operator via the Operator SDK:
 
 ```
-$ operator-sdk run --local --watch-namespace=""
+$ OPERATOR_NAMESPACE=managed-upgrade-operator operator-sdk run --local --watch-namespace=""
```

Comment:
The namespace is openshift-managed-upgrade-operator right?

Reply:
Seems to just follow what is created in the create project step -> `oc new-project managed-upgrade-operator`

Reply:
I think it would still work and be fine to mention some other namespace for the upgradeconfig resource to be present but perhaps even the repo docs can uniformly maintain the same namespace name as in the deployment of the operator https://github.com/openshift/managed-upgrade-operator/blob/nodekeeper_controller/deploy/namespace.yaml#L4

Reply:
Makes sense. I'll update that :+1: